packages/radix-ui-themes/src/helpers/props/text-align.prop.ts

```diff
@@ -4,7 +4,7 @@ const alignValues = ['left', 'center', 'right'] as const;
 
 const alignProp = {
   type: 'enum',
-  className: 'rt-r-align',
+  className: 'rt-r-ta',
```

Comment:
Found another broken classname
Looked through other props and haven't found anything suspicious, hope this one is the last